proposals/improved-interpolated-strings.md

```diff
@@ -224,15 +251,15 @@ public ref struct InterpolatedStringBuilder
         ArrayPool<char>.Shared.Return(_array);
         Return result;
     }
-    public bool TryFormat(string s) => TryFormat((ReadOnlySpan<char>)s);
-    public bool TryFormat(ReadOnlySpan<char> s)
+    public bool TryFormatBaseString(string s) => TryFormatInterpolationHole((ReadOnlySpan<char>)s);
```

Comment:
Nit: the cast is no longer needed

Reply:
Not if we have a generic. `TryFormatInterpolationHole<T>(T)` will be preferred over `TryFormatInterpolationHole(ReadOnlySpan<char>)`

Reply:
That's ok.  It's just an optimization at that point.  The generic handed a string will work just fine.